server/postgresApp.js

```diff
@@ -0,0 +1,163 @@
+const express = require('express');
+const app = express();
+const pgres = require('../PostgreSQL/index.js');
+const path = require('path');
+
+
+app.use(express.json())
+app.use('/products', express.static('./client/dist'))
+app.use('/', express.static('./client/dist'))
+
+pgres.connect();
+
+
+app.get('/api/productOptions/products/:id', (req, res) => {
+  let builtResponse;
+  let id = req.params.id;
+  let getItemsQuery = `SELECT * FROM items WHERE id = ${id}`;
+  let getReviewsQuery = `SELECT * FROM reviews WHERE prodId = ${id}`;
+
+  // ideally this would be a single query that joins the reviews result as a property of the items table, but I have to move ahead for now
+  pgres.query(getItemsQuery)
+    .then(result => {
+      builtResponse = result.rows[0];
+      builtResponse.price = {originalPrice: builtResponse.originalPrice, salePrice: builtResponse.salePrice};
+      delete builtResponse.originalPrice;
+      delete builtResponse.salePrice;
+      return pgres.query(getReviewsQuery);
+    })
+    .then(result => {
+      builtResponse.reviews = result.rows;
+      return res.json(builtResponse);
+    })
+    .catch((err) => {
+      console.error(err);
+      return res.status(400).send({error: `Item with ID: ${id} not found`});
+    })
+})
+
+
+app.post('/api/productOptions/products/:id/reviews', (req, res) => {
+  const { overallRating, easeOfAssembly, valueForMoney, productQuality, appearance, worksAsExpected, header, body, createdAt, iRecommendThisProduct } = req.body;
```

Comment:
niiice use of destructuring! Later on maybe reformat air bnb style to make reading easier: 
const {
            overallRating,
            etc.
           }
but also up to you style-wise I think!

Reply:
this would get rid of my mysterious 'b's altogether from request body lol

Reply:
You're right, I should break those destructurings down to be on multiple lines. Good catch! Thanks!